File: app/src/main/java/com/shen/xi/android/tut/sinablog/TagResult.java
package com.shen.xi.android.tut.sinablog;


import com.google.api.client.util.Key;

import java.util.ArrayList;

public class TagResult {
    @Key
    String qjson;
    @Key
    public int cnt;
    @Key
    public ArrayList<String> list;
    @Key
    public boolean isLastPage;
}

File: app/src/main/scala/com/shen/xi/android/tut/sinablog/TagResult.scala
package com.shen.xi.android.tut.sinablog


import com.google.api.client.util.Key
import java.util

class TagResult {
  @Key
  var qjson: String = ""
  @Key
  var cnt: Int = 0
  @Key
  var list: util.ArrayList[String] = null
  @Key
  var isLastPage: Boolean = false
}
